lib/PuppeteerSharp/ContextPayload.cs

```diff
@@ -6,8 +6,17 @@ public class ContextPayload
         public ContextPayload(dynamic context)
         {
             Id = context.id;
+            AuxData = context.auxData.ToObject<AuxData>();
         }
 
-        public string Id { get; internal set; }
+        public int Id { get; internal set; }
+
+        public AuxData AuxData { get; internal set; }
+    }
+
+    public class AuxData
```

Comment:
1. This could be an `struct`
2. I would give it another name, something like `ContextPayloadData`. and move it to its own file.